shell/components/SortableTable/index.vue

```diff
@@ -512,7 +514,7 @@ export default {
   },
 
   created() {
-    this.debouncedRefreshTableData = debounce(this.refreshTableData, 500);
+    this.debouncedRefreshTableData = debounce(this.refreshTableData, 500, { leading: true });
```

Comment:
This forces us to trigger at the beginning of the timer instead of the end which makes the visual for loading appear sooner.